### DISCUSSION TITLE
我还是不知道怎么在vue里面用，有没有使用的demo

### DISCUSSION THREAD

**Discussion by zhuyanxuan** [walinejs/waline#458] [Q&A] [answered]
官网里面就是update deploy方法，看不太明白啊

**Comment by greenhat616** [comment]
这个需要服务端的….你看的那个是部署服务端的教程

**Comment by lizheming** [comment, marked as answer]
https://github.com/walinejs/waline/blob/main/docs/.vuepress/theme/components/Waline.ts#L19-L69 Vue 的你看看官网的例子
